server/weather.go

```diff
@@ -183,6 +194,8 @@ func Query(w http.ResponseWriter, r *http.Request) {
 func main() {
 	http.HandleFunc("/", Index)
 	http.HandleFunc("/query", Query)
+	// note that if you run locally, you'll have to change the port number, since Linux doesn't
+	// let you use ports below 1024
 	log.Println("Starting server on http://127.0.0.1:80")
-	log.Fatal(http.ListenAndServe(":80", nil))
+	log.Fatal(http.ListenAndServe(":3005", nil))
```

Comment:
This looks wrong for production. Also the comment is wrong. Linux does let you use ports < 1024, but you have to be a privileged user.

Reply:
@Fingel As mentioned in the PR, I left the debugging settings in so that the reviewers could easily test it, and before it gets merged, I'll change this line and the corresponding line in Site.vue to match the old port number.

Reply:
@Fingel Regarding the permissions comment, I was a bit unclear in how I worded it, I meant to say that you can't use ports under 1024 unless you use `sudo`. I'll rewrite the comment to clarify this -- thanks for the feedback.